core/toolbox/category.js

```diff
@@ -108,15 +115,17 @@ Blockly.ToolboxCategory = function(categoryDef, toolbox, opt_parent) {
   this.cssConfig_ = {
     'container': 'blocklyToolboxCategory',
     'row': 'blocklyTreeRow',
+    'rowContentContainer': 'blocklyTreeRowContentContainer',
     'icon': 'blocklyTreeIcon',
     'label': 'blocklyTreeLabel',
     'contents': 'blocklyToolboxContents',
     'selected': 'blocklyTreeSelected',
     'openIcon': 'blocklyTreeIconOpen',
-    'closedIcon': 'blocklyTreeIconClosed'
+    'closedIcon': 'blocklyTreeIconClosed',
   };
 
-  Blockly.utils.object.mixin(this.cssConfig_, categoryDef['cssConfig']);
+  var cssConfig = categoryDef['cssConfig'] || categoryDef['cssconfig'];
```

Comment:
Curious why this is needed?

Reply:
It's not needed just nice to have. We currently have json and xml for the toolbox definitions and so all of our definitions are lowercase, however I have accidentally done camel case a lot in the JSON when testing and figure developers will as well. So just adding it as an option.

Reply:
Is this something that can be normalized in the xml to json toolbox method? That way we're not checking for both cases all over the place?

Reply:
Not the way it currently is. We only go through the given toolbox definition if it is not the correct format (ie xml, Array<Node>, string). If it is JSON we just return it. I could create a validate method to go through all of the JSON to validate it but I am unsure if this is worth it or not.

Reply:
Ok. Good to keep that option available, but yeah not worth it at this point.